push-to-docker.sh

```diff
@@ -1,5 +1,3 @@
-docker tag rippled-build-bot stevenzeiler/rippled-build-bot:$CIRCLE_SHA
-docker tag rippled-build-bot stevenzeiler/rippled-build-bot:$1
+docker tag -f rippled-package-builder stevenzeiler/rippled-package-builder:$1
```

Comment:
In case there is already a build with that tag locally. Without `-f` it will exit without creating the tag even though changes are made.